helpers/mime.js

```diff
@@ -1,38 +1,16 @@
 'use strict';
 
+const mime = require('mime-types');
+
 /**
  * getMimeFromFileExtension
  * @param  {String} fileExtension  File extension (e.g. css, txt, html)
  * @return {String} text/html
  */
 function getMimeFromFileExtension(fileExtension) {
-    let mime = '';
-
-    switch (fileExtension.toLowerCase()) {
-    case 'html':
-        mime = 'text/html';
-        break;
-    case 'css':
-        mime = 'text/css';
-        break;
-    case 'js':
-        mime = 'text/javascript';
-        break;
-    case 'png':
-        mime = 'image/png';
-        break;
-    case 'jpeg':
-        mime = 'image/jpeg';
-        break;
-    case 'jpg':
-        mime = 'image/jpeg';
-        break;
-    default:
-        break;
-    }
-
-    return mime;
+    return mime.lookup(fileExtension) || '';
```

Comment:
might as well do a fast bail of the mime lookup is empty/falsey here at https://github.com/screwdriver-cd/store/blob/ccc9af340566cc7a9ae253768ca0e7f71b5276bf/plugins/builds.js#L106-L116